### PR TITLE
fix bug with feeRemainder

### PR DESCRIPTION
**Pull request by decadanse**
removed (which was in comments until last commit) last test, need to move it into new context in 16-branch. thus coverage will be 100%

closes #28 

probably it's better to solve merge conflicts closer to the end of merging branches, because for now, main-branch-code locally crashes tests of this branch.

### CODE REVIEW

File: test/seed.js [hunk 1]
```diff
@@ -427,6 +447,45 @@ describe("Contract: Seed", async () => {
...
+            let classFee = parseEther("0.44").toString(); //SECOND_CLASS_FEE
+            let currentBuyAmount = expectedFeeAmount * classPrice / classFee + ethers.BigNumber.from("1");
+            console.log("JS currentBuyAmount %s \n", currentBuyAmount);
+
+            let { ["1"]: feeAmount } = await setup.seed
+                .connect(buyer4)
+                .callStatic.buy(currentBuyAmount.toString());
+            expect((await feeAmount).toString()).to.equal(getSeedAmounts("0"));
+
+            /*-------- correct version with corret result. but it breaks other tests below.
```
Comment (BritikovKI) [inline test/seed.js:472]: Then remove it, or rewrite it that way so it won't break other tests

File: contracts/seed/Seed.sol [hunk 2]
```diff
@@ -397,15 +404,25 @@ contract Seed {
...
+        console.log("classes[funders[msg.sender].class].classFee %s",classes[funders[msg.sender].class].classFee);
+        console.log("PRECISION %s",PRECISION);
+
 
         // seed amount vested per second > zero, i.e. amountVestedPerSecond = seedAmount/vestingDuration
         require(
             seedAmount >= vestingDuration,
             "Seed: amountVestedPerSecond > 0"
         );
+        // console.log("_fundingAmount %s",_fundingAmount);
```
Comment (BritikovKI) [inline contracts/seed/Seed.sol:423]: No need in commented out logs

File: test/seed.js (hunk 1 above)
Comment (decadanse) [reply test/seed.js]: Removed

File: contracts/seed/Seed.sol (hunk 2 above)
Comment (decadanse) [reply contracts/seed/Seed.sol]: logs cleared

**Review by BritikovKI — APPROVED** [review]

**Review by arseneeth — APPROVED** [review]

**Pull request merged**